prisma/seed-scheduled-times.ts

```diff
@@ -0,0 +1,80 @@
+import { PrismaClient, RequestStatus } from "@prisma/client";
+
```

Comment:
Opinion: now that we have submission form working - we can create appointments using the app itself. Right now, a user can create multiple requests under one id, and we can use that for testing. 

I am thinking we don't need this seeding code anymore.

Reply:
i did not update that file. that must be a vestige